spec/models/lesson_completion_spec.rb

```diff
@@ -2,24 +2,22 @@
 
 describe LessonCompletion do
 
-  let!(:user){ FactoryGirl.create(:user) }
-  let!(:lesson){ FactoryGirl.create(:lesson) }
+  let!(:user)  { double("User",    :id => 1) }
+  let!(:lesson){ double("Lessons", :id => 1) }
```

Comment:
Looking at this more closely, can we remove these two lets and just hard code the id's in the subject please? 😄